packages/views.py

```
@@ -3,13 +3,16 @@
 from rest_framework.response import Response
 
 from packages.modules import npm
-from packages.serializers import VersionedPackageSerializer
+from packages.serializers import PackageSerializer
 
 
 class PackageView(views.APIView):
     renderer_classes = [renderers.JSONRenderer]
 
-    def get(self, request: Request, package_name: str, version: str):
-        package_info = npm.get_versioned_package(package_name, version)
-        serializer = VersionedPackageSerializer(package_info)
+    def get(self, request: Request, package_name: str, range: str | None = None):
+        if range is None:
```

in case of packages like @snyk/snyk-docker-plugin -> range will not be an appropriate version. Need to verify range indeed has expected expression pattern (you can use regular expression), and throw exception in case of unexpected range pattern.